R/spatial_vfold_cv.R

```diff
@@ -230,15 +239,14 @@ posthoc_buffer_rset <- function(data,
 
   split_objs <- tibble::tibble(
     splits = split_objs,
-    id = names0(length(split_objs), "Fold"),
     v = v
   )
 
   split_objs$splits <- map(split_objs$splits, rm_out, buffer = buffer)
 
   new_rset(
     splits = split_objs$splits,
-    ids = split_objs[, grepl("^id", names(split_objs))],
+    ids = rset[, grepl("^id", names(rset))],
```

Comment:
@hfrick Is this the best way to grab all the columns, other than `splits`, of an rset object returned by rsample?

This is an internal function, which is taking `rset` straight from an rsample function, so there's no chance a user has renamed columns or anything else.

Reply:
that looks good to me 👍